src/style.css

```diff
@@ -0,0 +1,106 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
+
+.card {
+  height: 406px;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__logo {
+  height: 134px;
+  width: 160px;
+  margin: 32px 19px 0;
+}
+
+.card__title {
+  margin: 36px 16px 0;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__product-code {
+  font-size: 10px;
+  line-height: 14px;
+  margin: 4px 16px 0;
```

Comment:
do not use horizontal margins in such cases, just set paddings for `card`, make sure to fix it everywhere